docs/domain-versions.md

```diff
@@ -2,14 +2,15 @@
 
 The convention used by this document is to explain paths in context of domain
 "example-en", and with files starting from the bundle root. So the file EKN_VERSION would
-be listed as `/share/ekn/data/example-en/EKN_VERSION`.
+be listed as `/share/ekn/data/com.endlessm.example.en/EKN_VERSION`.
 
 Since knowledge-lib is part of the core OS, and is updated separately from application
 bundles, this means that knowledge-lib needs some form of backwards compatibility with
 bundles, even if we change parts of the bundles.
 
 A set of content is called a "domain". The format of content shipped in a domain is
-specified with a file, `/share/ekn/data/example-en/EKN_VERSION`, shipped in each bundle,
+specified with a file, `/share/ekn/data/com.endlessm.example.en/EKN_VERSION`,
+shipped in each bundle,
 which contains a number. If it's missing, it's assumed to be a version 1 domain.
 
 Domains have two major parts: 1. a Xapian database of records which is used for finding
```

Comment:
Should we remove text about V1/V2 now that the code doesn't support them anymore?